### ISSUE TITLE
Keyboard nonresponsive on steam link hardware with moonlight, works fine with steamlink.

### ISSUE THREAD

**Issue by Zod1234** [moonlight-stream/moonlight-qt#1428]
**Describe the bug**
I have a steam link hardware device.  a basic keyboard (logitech k120) wired hooked up to the steam link.  In the steamlink interface the keyboard works without issue and when connecting to my pc via steamlink.  It also works in the moonlight app, but as soon as I connect, the keyboard is nonfunctional.  

**Affected games**
All games and desktop.  No keyboard functionality.

**Moonlight settings (please complete the following information)**
moonlight settings at default.

**Client PC details (please complete the following information)**
 - OS: Steam link hardware
 - Moonlight Version:  v6.1.0
 
 
**Server PC details (please complete the following information)**
 - OS: Win 11
 - Sunshine or GeForce Experience version: sunshine Version v0.23.1
 - GPU: RTX 4070
 - GPU driver: 561.09

**Moonlight Logs (please attach)**
- don't know how to get logs from steam link hardware.

**Additional context**
As mentioned, I don't have a keyboard plugged in on host.  This is not an issue when connecting with steam link, only with moonlight.  Not sure if sunshine or moonlight needs the host to have a keyboard.  Steam link needs the host to have a mouse, but not a keyboard.  In the moonlight app on steam link, I can navigate the menu with just the keyboard, but as soon as I connect it's not responsive.  Tried hotkey combos for moonlight to no effect.

**Comment by jorys-paulin** [comment]
I've been able to reproduce this issue on my Steam Link.

Everything worked as expected until I updated my Steam Link to beta 887.
Now, keyboard and mouse don't work as soon as streaming starts, including Moonlight keyboard shortcuts, until I physically unplug then plug them back in. My controller still worked as expected, though.
While the streaming was starting, all of them seemed to disconnect then reconnect at random, including when starting Moonlight itself.
Here is what I could grab from `/tmp/moonlight.log` on build 887:

```
# cat /tmp/moonlight.log
linux_version_check - failed to open /proc/mv88de3010
linux_version_check - app expects linux kernel revision 6154, please manually check current revision
<<<<< CPU-1 OSAL Init
memory map size for cache:0x0C000000, fd_cache:7, ret:0
memory map size for non-cache:0x02000000, fd_noncache:6
MV_SHM_Init OK:
base_virtaddr_cache:0xA5B60000, size:0x0C000000
base_virtaddr_noncache:0xA3B60000, size:0x02000000
DBG HEALTH {line: 280 (MV_CC_RPCClnt_Call)} 0x7
DBG HEALTH {line: 280 (MV_CC_RPCClnt_Call)} 0x7
DBG HEALTH {line: 1072 (MV_PE_Remove)} 0x80000018
DBG HEALTH {line: 280 (MV_CC_RPCClnt_Call)} 0x8
DBG HEALTH {line: 1199 (MV_CC_RPCSrv_RecvTask)} 0x1
DBG HEALTH {line: 1042 (MV_CC_RPCSrv_Task)} 0x1
```

I've rolled-back to public 883 and everything works again, but the keyboard, mouse and gamepad still seem to disconnect while in the starting/stopping stream view.
Here is what I could gather from `/tmp/moonlight.log` while on build 883:

```
# cat /tmp/moonlight.log
linux_version_check - failed to open /proc/mv88de3010
linux_version_check - app expects linux kernel revision 6154, please manually check current revision
<<<<< CPU-1 OSAL Init
memory map size for cache:0x0C000000, fd_cache:7, ret:0
memory map size for non-cache:0x02000000, fd_noncache:6
MV_SHM_Init OK:
base_virtaddr_cache:0xA5B44000, size:0x0C000000
base_virtaddr_noncache:0xA3B44000, size:0x02000000
DBG HEALTH {line: 280 (MV_CC_RPCClnt_Call)} 0x7
DBG HEALTH {line: 280 (MV_CC_RPCClnt_Call)} 0x7
DBG HEALTH {line: 1072 (MV_PE_Remove)} 0x80000017
DBG HEALTH {line: 280 (MV_CC_RPCClnt_Call)} 0x8
DBG HEALTH {line: 1199 (MV_CC_RPCSrv_RecvTask)} 0x1
DBG HEALTH {line: 1042 (MV_CC_RPCSrv_Task)} 0x1
```

I've used the link's "save debug logs" to get the attached log file, actually that time streaming failed completly, black screen and five seconds later a "no video received from host". The second try worked though.
[system_debug.log](https://github.com/user-attachments/files/17178644/system_debug.log)

**Comment by Zod1234** [comment]
I feel like an idiot.  I don't know why I didn't think to roll back from the beta.  I forgot I updated to the beta.  I rolled back and it's working.  Strange we're getting different issues though.  My mouse always worked, it was just keyboard.  For you it was both.  I've used the stable version for a few hours and no issues so far, not with devices anyway.

I do get a bit of audio scratchiness/jitters with moonlight on my steamlink that I don't get with steam link streaming.  I read packet loss can cause this but I'm not getting any packet loss and i've tried much lower bitrates but the scratchiness is still there even though the video is perfect.  But that's a different issue and hopefully there's a way around that.

I've just loaded up virtualhere and have that working with steamlink, but I'm gathering this doesn't work with moonlight.  That's unfortunate if correct.  Searching for workarounds for that too.
